src/webgpu/shader/validation/wgsl/duplicate-var-in-one-scope-v2.fail.wgsl

```diff
@@ -0,0 +1,9 @@
+// v-0014 - This fails because variable `i` is redeclared
```

Comment:
Nit: The WGSL spec no longer has validation error code, so please remove the "v-0014" from these tests.

Reply:
done